docs/getting-started/install_capi_operator.md

```diff
@@ -4,4 +4,57 @@ sidebar_position: 3
 
 # Install CAPI Operator
 
-This section needs to be filled out.
\ No newline at end of file
+## Installing CAPI and providers
+
+There are two options when it comes to installing [CAPI](https://github.com/kubernetes-sigs/cluster-api). You can install CAPI and desired providers with [clusterctl](https://cluster-api.sigs.k8s.io/user/quick-start#install-clusterctl), following the official [quick-start](https://cluster-api.sigs.k8s.io/user/quick-start#installation) guide, or you can use helm based installation with [Cluster API Operator](https://github.com/kubernetes-sigs/cluster-api-operator).
```

Comment:
As this section is installed CAPI operator then i think we can just cover that and not mention the clusterctl path at all